src/test/resources/config/application.yml

```diff
@@ -78,7 +78,9 @@ artemis:
                     default: "~~invalid~~"
         build-agent:
             short-name: "artemis-build-agent-test"
-
+    telemetry:
+        enabled: false  # setting this to false will disable sending any information to the telemetry service
+        
```

Comment:
Is this change intentional for this PR? You also make this change in `src/test/resources/config/application-artemis.yml`

Reply:
Yes, although I could have done that in a different PR. I noticed that during integration test startup, there are exceptions, as the tests start the telemetry service, which tries to send data to localhost, which obviously does not work. 
This basically just disables the telemetry service spamming logs for tests.
It doesn't affect the tests for the Telemetry service itself, as there the profiles and variables needed are configured manually anyway. 

I added this change here, because I felt the change was too minor, to open a separate PR just for that, tbh.